dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/FATSuite.java

```diff
@@ -33,18 +40,38 @@
 
 public class FATSuite extends TestContainerSuite {
 
-    public static RepeatTests testRepeatMPTel20(String serverName) {
-        return RepeatTests.with(FeatureReplacementAction.EE11_FEATURES())
-                        .andWith(FeatureReplacementAction.EE10_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE9_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly())
-                        .andWith(FeatureReplacementAction.EE7_FEATURES().fullFATOnly());
-
+    public static RepeatTests allMPRepeatsWithMPTel20OrLater(String serverName) {
+        return TelemetryActions
+                        .repeat(serverName, MicroProfileActions.MP70_EE11, MicroProfileActions.MP70_EE11_APP_MODE, MicroProfileActions.MP70_EE10,
+                                TelemetryActions.MP50_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP14_MPTEL20);
     }
 
     public static RepeatTests testRepeatMPTMetrics5(String serverName) {
         return RepeatTests.with(FeatureReplacementAction.EE11_FEATURES())
                         .andWith(FeatureReplacementAction.EE10_FEATURES().fullFATOnly());
 
     }
+
+    //If we're in the app mode repeat this returns an updated Web Archive, otherwise it sets environment properties
+    public static WebArchive setTelProperties(WebArchive archive, LibertyServer server) {
+        if (RepeatTestFilter.isRepeatActionActive(MicroProfileActions.MP70_EE11_APP_MODE.getID())) {
+            return archive.addAsManifestResource(new File("publish/resources/META-INF/microprofile-config.properties"),
+                                                 "microprofile-config.properties");
+        } else {
+            server.addEnvVar("OTEL_METRICS_EXPORTER", "otlp");
+            server.addEnvVar("OTEL_SDK_DISABLED", "false");
+            server.addEnvVar("OTEL_TRACES_EXPORTER", "none");
+            server.addEnvVar("OTEL_LOGS_EXPORTER", "none");
+            server.addEnvVar("OTEL_METRIC_EXPORT_INTERVAL", "200");
+            return archive;
+        }
+    }
+
+    public static String getAppNameOrUnkownService(String appName) {
+        if (RepeatTestFilter.isAnyRepeatActionActive(MicroProfileActions.MP70_EE11_APP_MODE.getID())) {
+            return appName;
+        } else {
+            return "unkown_service";
```

Comment:
```suggestion
            return "unknown_service";
```
Typo here, but not sure if it's supposed to match a typo elsewhere....

Reply:
It will fail with the correct spelling, but we'd better fix that too. Opening another PR.